tests/resource/test_resource_sub_function_requests.py

```diff
@@ -0,0 +1,163 @@
+"""Test general resource profile setter functions."""
+
+import unittest
+from unittest.mock import Mock
+
+import __init__
+
+import tests.resource.test_resource_constants as TestResourceConstants
+from pyracf import ResourceAdmin
+from pyracf.common.irrsmo00 import IRRSMO00
+
+# Resolves F401
+__init__
+
+
+class TestResourceSubFunctionRequests(unittest.TestCase):
```

Comment:
"Subfunction" should be one word. Changes file names and code to make "subfunction" one word.

Reply:
Sure; done.

Reply:
Please update the filenames as well.

Reply:
done;